views/account/settings.hbs

```diff
@@ -1,213 +1,203 @@
 {{#extend "lib/loggedin"}}
```

Comment:
please double check this file, as here are large diffs

Reply:
checked, small fixes added, work correct now